pkgs/by-name/li/lianad/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lianad";
+  version = "5.0"; # keep in sync with liana
+
+  src = fetchFromGitHub {
+    owner = "wizardsardine";
+    repo = "liana";
+    rev = "v${version}";
+    hash = "sha256-RkZ2HSN7IjwN3tD0UhpMeQeqkb+Y79kSWnjJZ5KPbQk=";
+  };
+
+  cargoSha256 = "sha256-6r8G4sO/aDO5VyJbmbdFz3W2b8ZmulXJZtnz8IAwcaU=";
+
+  postInstall = ''
+    install -Dm0644 ./contrib/lianad_config_example.toml $out/etc/liana/config.toml
+  '';
+
+  doCheck = false;
+
+  meta = with lib; {
+    mainProgram = "lianad";
+    description = "A Bitcoin wallet leveraging on-chain timelocks for safety and recovery";
+    homepage = "https://wizardsardine.com/liana";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ dunxen plebhash ];
```

Comment:
If you add yourself in as a maintainer, you also have to have an entry for yourself, `plebhash` in https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix

Reply:
done